src/PostgREST/Config.hs

```diff
@@ -112,6 +113,7 @@ readOptions = do
           <*> (fromMaybe 3000 . join . fmap coerceInt <$> C.key "server-port")
           <*> (fmap encodeUtf8 . mfilter (/= "") <$> C.key "jwt-secret")
           <*> (fromMaybe False . join . fmap coerceBool <$> C.key "secret-is-base64")
+          <*> (fromMaybe "" . mfilter (/= "") <$> C.key "jwt-aud")
```

Comment:
I'm thinking the mfilter is not necessary here since all it would do is convert `Just ""` to `Nothing` which `fromMaybe` would return as `""`.